src/main/kotlin/com/sourcegraph/cody/ui/HtmlViewer.kt

```diff
@@ -32,14 +25,4 @@ object HtmlViewer {
     jEditorPane.addHyperlinkListener(BrowserHyperlinkListener.INSTANCE)
     return jEditorPane
   }
-
```

Comment:
@mkondratek I think you added it, do you maybe remember if it was needed for any particular case?
According to my testing it rather seems to be unused (which might be good thing, because it would probably not recompute on theme switch anyway).
Screenshots from the PR are without that code.

Reply:
Something strange happened to the repository I do not recall changing this file. For some reason, git(-hub?) assigns me as the author of many changes.